src/components/CurrentLocationInfo.css

```diff
@@ -0,0 +1,11 @@
+.currentLocation {
```

Comment:
We tent to call `css` files `styles.css` but it's not a community rule, just internal thing